test/testmachinery/shoot/enable_rotate_certs_disable_test.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package shoot
+
+import (
+	"context"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
```

Comment:
Imported in the project as 
```suggestion
	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
```

This makes me think that we should also introduce the importas linter: https://github.com/gardener/gardener/blob/09bc51f5af12a9aeab2a8be146bf7601f556866e/.golangci.yaml.in#L69